src/Cms/Language.php

```diff
@@ -571,7 +580,7 @@ public function update(array|null $props = null): static
 		$updated = $this->clone($props);
 
 		if (isset($props['translations']) === true) {
-			$updated->translations = $props['translations'];
+			$updated->translations = new LanguageTranslations($updated, $props['translations']);
```

Comment:
Does this work? I thought `translations` property is protected.

Reply:
It was already protected: `protected array $translations`?